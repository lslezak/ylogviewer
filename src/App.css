body {
  margin: 2em;
}

.smaller_font {
  font-size: 80%;
}
