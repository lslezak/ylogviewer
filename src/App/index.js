import React, { Component } from 'react';
import { Alert, AlertActionCloseButton } from '@patternfly/react-core';
import './app.css';

export default class App extends Component {
  state = {
    isShowing: true
  };
  dismissNotification = () => {
    this.setState({ isShowing: false });
  };
  render() {
    const { isShowing } = this.state;
    return (
      <div className="app-container">
        {isShowing && (
          <div className="notification-container">
            <Alert
              variant="success"
              title="Setup Complete"
              actionClose={<AlertActionCloseButton onClose={this.dismissNotification} />}
            >
              You have successfully launched your patternfly starter project.
            </Alert>
          </div>
        )}
      </div>
    );
  }
}
