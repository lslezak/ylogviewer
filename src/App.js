
import React, { useState, useEffect } from "react";

import NotesCard from "./NotesCard";
import InputSelectionCard from "./InputSelectionCard";
import PageHeader from "./PageHeader";

import "./App.css";

export default function App() {
  // the notes are visible
  const [isOpen, setIsOpen] = useState(true);

  const close = () => {
    setIsOpen(false);
    if (onCloseCallback) onCloseCallback();
  };

  return (
    <>
      <header>
        <PageHeader/>
      </header>
      <main>
        <InputSelectionCard/>
        <br />
        <NotesCard />
      </main>
    </>
  );
};
