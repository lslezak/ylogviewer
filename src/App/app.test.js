import React from 'react';
import { shallow } from 'enzyme';
import App from './';

describe('App tests', () => {
  test('should render default App component', () => {
    const view = shallow(<App />);
    expect(view).toMatchSnapshot();
  });
});
