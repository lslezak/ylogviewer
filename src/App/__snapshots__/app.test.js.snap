// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests should render default App component 1`] = `
<div
  className="app-container"
>
  <div
    className="notification-container"
  >
    <Alert
      actionClose={
        <AlertActionCloseButton
          onClose={[Function]}
        />
      }
      title="Setup Complete"
      variant="success"
    >
      You have successfully launched your patternfly starter project.
    </Alert>
  </div>
</div>
`;
